src/scripts.js

```diff
@@ -36,13 +44,44 @@ const showProfile = () => {
   `)
 }
 
+//ADD ALL-TIME SLEEP STATS:
+
 //loads a display of the user's step goal
 //also loads display of community step goal
 const showStepGoalComparison = () => {
   userStepGoal.insertAdjacentHTML('beforeend', `<p>${user.dailyStepGoal}</p>`)
   communityStepGoal.insertAdjacentHTML('beforeend', `<p>${community.findAverageStepGoal()}</p>`)
 }
 
+//ADD TODAY's HYDRATION STATS:
+const showHydrationStats = () => {
+  waterStats.insertAdjacentHTML('beforeend',
+    `<p class="water-stats">Water consumed today: ${communityHydration.calculateTotalWaterOnDay(hydration.userID, "2019/09/20")} OZ</p>
+    `)
+}
+
+//ADD WEEKLY HYDRATION STATS:
```

Comment:
Since the insertion is coming 'beforeend' and somehow we put friends div inside of graph, this isn't getting populated inside the div where we want it.

 We could also add parameters for that start and end date when calling the showHydrayionStatsWeek on pageLoad or whenever we call it, so we don't have to do it over and over in the calculateTotalWeek() call.

Reply:
@srslie That makes sense.  And we might need to adapt to whatever we ultimately end up using to display these graph results.

Reply:
@srslie I think I've updated everything by now, so let me know if I'm missing anything! Otherwise, we should be good to merge.